PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+## Description
+<!-- Please explain the changes you made here. -->
+
+## Testing done
+<!-- Please explain local testing done, this can include
+automated testing, manual testing, cross browser testing -->
+
+## Testing Plan
+<!-- Please explain testing planned this can include
+automated testing, manual testing, cross browser testing -->
+
+## Screenshots
+<!-- Please provide screenshots of UI changes if applicable -->
+
+## Acceptance Criteria (Definition of Done)
+
+#### Unique to this PR
+<!-- This would be a good place to include feature flag check item and info, specific dashboards and instrumentation check item and info -->
+- [ ] _Replace this line with individual tasks unique to your PR_
+
+#### Applies to all PRs
+
+- [ ] Appropriate logging
+- [ ] Swagger docs have been updated, if applicable
```

Comment:
Vets-website doesn't use swagger for documentation, and documentation is housed in several locations (e.g. `https://github.com/usds/us-forms-system/tree/master/src/js` and the `/docs` folder on this repo), should we list those (perhaps as a checklist) instead here or simply say `update documentation`?

Reply:
Thanks for pointing this out. I've updated the checklist item to be generic - documentation has been updated.